src/main/service/listParseHelperService.ts

```diff
@@ -188,24 +188,28 @@ export class ListParseHelperService {
      * @param session
      */
     public findAndManipulateJudiciary(session: object): string {
-        let judiciaries = '';
-        let foundPresiding = false;
+        const judiciaries = [];
         session['judiciary']?.forEach(judiciary => {
-            if (judiciary?.isPresiding === true) {
-                judiciaries = ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs);
-                foundPresiding = true;
-            } else if (!foundPresiding) {
-                if (ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) !== '') {
-                    judiciaries += ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) + ', ';
-                }
-            }
+            const name = ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs);
+            judiciary?.isPresiding === true ? judiciaries.unshift(name) : judiciaries.push(name);
         });
+        return judiciaries.join(', ');
+    }
 
-        if (!foundPresiding) {
-            judiciaries = judiciaries.slice(0, -2);
-        }
-
-        return judiciaries;
+    /**
+     * Manipulate crime list judiciary data for writing out to screen.
+     * @param session
+     */
+    public findAndManipulateJudiciaryForCrime(session: object): string {
+        const judiciaries = [];
+        session['judiciary']?.forEach(judiciary => {
+            const name =
+                ListParseHelperService.writeStringIfValid(judiciary?.johTitle) +
+                ' ' +
+                ListParseHelperService.writeStringIfValid(judiciary?.johNameSurname);
```

Comment:
The title could be blank and you don't want to add that extra space in that case. Better to do it like below when one of the fields is blank, it is not adding the space character:

const title = ListParseHelperService.writeStringIfValid(judiciary.johTitle);
const surname = ListParseHelperService.writeStringIfValid(judiciary.johNameSurname);
[title, surname].filter(j => j.length > 0).join(' ');